README.md

```diff
@@ -1 +1,250 @@
-# cli
\ No newline at end of file
+<div align="center">
+  <img alt="go-octopusdeploy Logo" src="https://user-images.githubusercontent.com/71493/133961475-fd4d769f-dc32-4723-a9bd-5529c5b12faf.png" height="140" />
+  <h3 align="center">cli</h3>
+  <p align="center">Command Line Interface for <a href="https://octopus.com/">Octopus Deploy</a> 🐙</p>
+  <p align="center">
+    <a href="https://github.com/OctopusDeploy/cli/releases/latest"><img alt="GitHub release" src="https://img.shields.io/github/v/release/OctopusDeploy/cli.svg?logo=github&style=flat-square"></a>
+    <a href="https://goreportcard.com/badge/github.com/OctopusDeploy/cli"><img src="https://goreportcard.com/badge/github.com/OctopusDeploy/cli" alt="Go Report"></a>
+  </p>
+</div>
+
+---
+
+## Overview
+
+This project aims to create a new CLI for communicating with the Octopus Deploy Server, written in Go.
```

Comment:
This project aims to create a new CLI (written in Go) for communicating with Octopus Deploy.